reco_utils/dataset/pandas_df_utils.py

```diff
@@ -74,3 +75,75 @@ def filter_by(df, filter_by_df, filter_by_cols):
             filter_by_df.set_index(filter_by_cols).index
         )
     ]
+
+
+def df_to_libffm(df, col_rating=DEFAULT_RATING_COL, filepath=None):
+    """Converts an input Dataframe (df) to an text file in libffm format.
```

Comment:
"a text file"

Reply:
Proofreading should be practiced every time submitting a PR. Should we put this into the coding principles 😄